src/Pantry.js

```diff
@@ -0,0 +1,59 @@
+class Pantry {
+  constructor(pantry) {
+    this.userPantry = pantry;
+    this.shoppingList = [];
+  }
+
+  hasAllIng(recipe) {
```

Comment:
The test approach may need to be abandoned based on how the logic will be written to check for the existence of ingredients at a later point